client/src/pages/LandingPage/index.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-import Header from '../../components/Header';
+import NavTap from '../../components/NavTap';
```

Comment:
Edit this component name here and the original one
```suggestion
import Navbar from '../../components/Navbar';
```